developer-docs-site/static/examples/typescript/tsconfig.json

```diff
@@ -7,7 +7,7 @@
     "types": [
       "node"
     ],
-    "module": "node12",
```

Comment:
Why this change?

Reply:
When I ran it on my mac book m1, it errored out saying `TSError: ⨯ Unable to compile TypeScript:
error TS6046: Argument for '--module' option must be: 'none', 'commonjs', 'amd', 'system', 'umd', 'es6', 'es2015', 'es2020', 'es2022', 'esnext', 'node16', 'nodenext'` and I didn't understand what the correct value should be. Looking up here https://www.typescriptlang.org/tsconfig#module saw the suggestion and it worked. I don't know Typescript enough to say what should be the value there.

Reply:
Did you run yarn? If yes, may be make sure you have the right nodejs version installed. I just tried running this at HEAD on my laptop (also an M1) and it works fine.

Reply:
I tried compiling with commonjs and it works so may be it's fine to keep it updated.